src/pages/index.js

```diff
@@ -1,8 +1,10 @@
 import { authService } from '../js/auth.js';
 import { standardizeString } from '../js/library.js';
+import { TaskItem } from '../components/task-item/index.js';
 
 /** @typedef { import('../js/auth.js').UserData } User */
 
+console.log(TaskItem.name);
```

Comment:
probably want to start moving away from outputting info to the log but for now i think its fine just keep in mind

Reply:
Yeah... I wonder if ```TaskItem.name = TaskItem.name;``` would work if we just need ```TaskItem``` to be used.

Reply:
This is the workaround for the eslint error. I can set up the build process that removes all `console.log`'s